mentat/git_handler.py

```diff
@@ -27,6 +27,8 @@ def get_non_gitignored_files(path: Path) -> set[Path]:
                 text=True,
             ).split("\n"),
         )
+        # windows-safe check if p exists in path
+        if Path(os.path.join(path, p)).exists()
```

Comment:
Is there a reason to use os.path.join instead of path / p?

Reply:
No - I think copilot wrote it for me 🙈